profiles/common/modules/features/nexteuropa_dgt_connector/tmgmt_dgt_connector/tmgmt_dgt_connector.module

```diff
@@ -82,13 +101,10 @@ function _tmgmt_dgt_connector_workbench_store_request_languages_callback(&$form,
   global $user;
 
   $target_languages = array_filter($form_state['values']['target_language']);
-  $enforced_source_language = NULL;
-  if ($form_state['values']['enforced_source_language']) {
-    $enforced_source_language = $form_state['values']['source_language'];
-  }
-
+  $enforced_source_language = $form_state['values']['enforced_source_language'] ? $enforced_source_language = $form_state['values']['source_language'] : NULL;
```

Comment:
This statement looks weird, shouldn't it be $enforced_source_language = $form_state['values']['enforced_source_language'] ? $form_state['values']['source_language'] : NULL;